classes/Link.php

```diff
@@ -820,6 +821,28 @@ public function getAdminLink($controller, $withToken = true, $sfRouteParams = ar
         return $this->getAdminBaseLink() . basename(_PS_ADMIN_DIR_) . '/' . Dispatcher::getInstance()->createUrl($controller, $idLang, $params);
     }
 
+    /**
+     * Adapter to get Admin HTTP link.
+     *
+     * @param string $controller the controller name
+     * @param bool $withToken
+     * @param array[string] $extraParams
+     *
+     * @return string
+     */
+    public function getLegacyAdminLink($controller, $withToken = true, $extraParams = array())
```

Comment:
do you need this new method? do you actually use it?

Reply:
not anymore => to be removed